docsrc/Makefile

```diff
@@ -17,4 +17,6 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
+	rm -rf ../docs
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	mv ./build/html ../docs
```

Comment:
Could just build it directly into `../docs`? 🤷‍♂️

Reply:
Doesn't appear possible... :-(